packages/mdx-deck/lib/static/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "mdx-deck",
```

Comment:
This seems like content that should be user-defined – this and the PWA-related code and icons probably should be in a separate issue/PR